package.json

```
@@ -20,6 +20,9 @@
   "bin": {
     "npcheck": "index.js"
   },
+  "engines": {
+    "node": "^16 || ^14"
```

Is there something that say anything higher than 14? I don't think we want to block running on newer versions?

---

npcheck is using some JS features that are not available on node 12.x, for example `?.`  optional chaining.

---

I think we could say 
```suggestion
    "node": ">=14"
```
FWIW `npm` doesn't block by default (I think it does warn). `yarn` (or at least some versions of `yarn`) did block unless a command line flag was passed.

---

`"node": ">=14"` looks better than (each) explicit version